examples/bpk-animate-height/examples.js

```diff
@@ -18,7 +18,8 @@
 
 /* @flow strict */
 
-import React, { Component, type Node } from 'react';
+import { Component } from 'react';
+import type { Node } from 'react';
```

Comment:
Can we not keep these combined as before and just remove the React import?

I know this was auto completed by the script but wondering if we can keep the change and the number of imports low. This will be a similar comment throughout so keeping to this one thread.

If it doesn't work then that is fine just want to check first

Reply:
Yep so this was done automatically by the scripts. I did try to see if it's possible to keep them combined and it doesn't seem to work. Also had a look at other team's PRs and it seems like that's how it is used in other repos in the company too